internals/review_models.py

```diff
@@ -334,7 +334,8 @@ def get_activities(cls, feature_id: int, gate_id: Optional[int]=None,
     query = Activity.query().order(Activity.created)
     query = query.filter(Activity.feature_id == feature_id)
     if gate_id:
-      query = query.filter(Activity.gate_id == gate_id)
+      # Fetch activities with a gate_id or None value.
+      query = query.filter(Activity.gate_id.IN([gate_id, None]))
```

Comment:
Cool.  I didn't know that we could use `None` with the `IN` operator.

Reply:
I didn't know either. I think ndb set the non-existing field value to None so I tried the None value. It appears that it worked based on the tests